src/app.module.ts

```diff
@@ -1,4 +1,4 @@
-import { Module } from '@nestjs/common';
+import { Logger, Module } from '@nestjs/common';
```

Comment:
**suggestion**: Consider adding a space after the comma for consistency in the import statement.